packages/web/src/containers/token-swap-container/TokenSwapContainer.tsx

```diff
@@ -1,56 +1,196 @@
+  swapRateUSD: 1.14,
+  priceImpact: 0.3,
+  guaranteedAmount: {
+    amount: 45124,
+    currency: "GNOT",
+  },
+  gasFee: {
+    amount: 0.000001,
+    currency: "GNOT",
+  },
+  gasFeeUSD: 0.1,
+};
+
+const swapTokenInfo: SwapTokenInfo = {
+  tokenA: {
+    chainId: "test3",
+    address: "0x111111111117dC0aa78b770fA6A738034120C302",
+    path: "gno.land/r/demo/1inch",
+    name: "1inch",
+    symbol: "1INCH",
+    decimals: 6,
+    logoURI:
+      "https://assets.coingecko.com/coins/images/13469/thumb/1inch-token.png?1608803028",
+    priceId: "1inch",
+    createdAt: "1999-01-01T00:00:01Z",
+  },
+  tokenAAmount: "0",
+  tokenABalance: "0",
+  tokenAUSD: 0,
+  tokenAUSDStr: "0",
+  tokenB: {
+    chainId: "test3",
+    address: "0x111111111117dC0aa78b770fA6A738034120C302",
+    path: "gno.land/r/demo/1inch",
+    name: "1inch",
+    symbol: "1INCH",
+    decimals: 6,
+    logoURI:
+      "https://assets.coingecko.com/coins/images/13469/thumb/1inch-token.png?1608803028",
+    priceId: "1inch",
+    createdAt: "1999-01-01T00:00:01Z",
+  },
+  tokenBAmount: "0",
+  tokenBBalance: "0",
+  tokenBUSD: 0,
+  tokenBUSDStr: "0",
+  direction: "EXACT_IN",
+  slippage: 10,
+};
+
+const FROM_DATA = {
+  token: {
+    chainId: "dev",
+    createdAt: "2023-10-10T08:48:46+09:00",
+    name: "Gnoswap",
+    address: "g1sqaft388ruvsseu97r04w4rr4szxkh4nn6xpax",
+    path: "gno.land/r/gnos",
+    decimals: 4,
+    symbol: "GNOS",
+    logoURI: "https://s2.coinmarketcap.com/static/img/coins/64x64/5994.png",
+    priceId: "gno.land/r/gnos",
+  },
+  amount: "0",
+  price: "$0.00",
+  balance: "0",
+};
+
+const TO_DATA = {
+  token: {
+    chainId: "dev",
+    createdAt: "2023-10-10T08:48:46+09:00",
+    name: "Gnoswap",
+    address: "g1sqaft388ruvsseu97r04w4rr4szxkh4nn6xpax",
+    path: "gno.land/r/gnos",
+    decimals: 4,
+    symbol: "GNOS",
+    logoURI: "https://s2.coinmarketcap.com/static/img/coins/64x64/5994.png",
+    priceId: "gno.land/r/gnos",
+  },
+  amount: "0",
+  price: "$0.00",
+  balance: "0",
+};
 
 const TokenSwapContainer: React.FC = () => {
-  const router = useRouter();
+  const [toData, setToData] = useState(TO_DATA);
+  const [fromData, setFromData] = useState(FROM_DATA);
+  const [copied, setCopied] = useState(false);
+  const [openedConfirmModal, setOpenedConfirmModal] = useState(false);
+  const [openedSetting, setOpenedSetting] = useState(false);
 
+  const themeKey = useAtomValue(ThemeState.themeKey);
   const swapNow = useCallback(() => {
-    router.push("/swap?from=GNOT&to=GNOS");
-  }, [router]);
+    setOpenedConfirmModal(true);
+  }, []);
 
   const connectWallet = useCallback(() => {
     console.log("Request Adena");
   }, []);
 
+  const handleSwap = () => {
+    setFromData(toData);
+    setToData(fromData);
+  };
+
+  const handleCopied = async () => {
+    try {
+      await navigator.clipboard.writeText(window.location.href);
+      setCopied(true);
+      setTimeout(() => {
+        setCopied(false);
```

Comment:
What does calling setCopied every 2 seconds mean?

Reply:
Just want to dispay tooltip 2 seconds @harryoh

Reply:
I think calling it every 2 seconds for a tooltip is too inefficient.
Is there any other way?

Reply:
It just show tooltip one time after click @harryoh

Reply:
my mistake. I mis-read it as `setInterval`.
no problems. :)